test/machine_test.go

```diff
@@ -18,26 +20,86 @@ var (
 	lookup = map[string]map[string]expected{
 		"happyTests":           happyTests,
 		"notIdentifiableTests": notIdentifiableTests,
-		"happyTestsOneOff":     happyTestsOneOff,
 	}
 	happyTests = map[string]expected{
```

Comment:
Hmm sorting isn't going to work once we get to 10, but I guess we can deal with that when/if we get there. This almost past acceptable hacky levels :D. We could change the `type dataObject map[string]testResource` from a map to a slice and we could randomize the things that can be random. But meh that feels like more change than necessary.

Reply:
doh, this should have been just a comment.